src/thirdparty/llvm-c/DataTypes.h

```diff
@@ -0,0 +1,90 @@
+/*===-- include/llvm-c/DataTypes.h - Define fixed size types ------*- C -*-===*\
```

Comment:
Are these llvm-c files headers necessary if we are only using the C++ API?

Reply:
Yes, some includes require them from elsewhere. (I just included necessary files as I saw them being used.)